xbmc/pvr/windows/GUIWindowPVRBase.cpp

```diff
@@ -359,7 +359,7 @@ void CGUIWindowPVRBase::SetInvalid()
 bool CGUIWindowPVRBase::CanBeActivated() const
 {
   // No activation if PVR is not enabled.
-  return CSettings::GetInstance().GetBool(CSettings::SETTING_PVRMANAGER_ENABLED);
```

Comment:
This was introduced to disable to open empty (ugly, maybe not working) pvr windows by for instance accidentally hitting the respective keyboard shortcut. This will not work anymore, now.

Either we fix the implementation or remove CGUIWindowPVRBase::CanBeActivated.

I opt for the first. Imo <code>return g_PVRClients->HasCreatedClients()</code> should do the trick.